chronoline/chronoline.js

```diff
@@ -394,13 +398,47 @@ function Chronoline(domElement, events, options) {
         // DRAWING
         t.floatingSet = t.paper.set();
         t.sectionLabelSet = t.paper.set();
+
+        //attach background click events function
+        t.myCanvas.onclick = function (e) {
```

Comment:
Yes you are probably right about this. I will update the pull request when I get time.